.changeset/khaki-cherries-appear.md

```diff
@@ -0,0 +1,5 @@
+---
+'formik': major
```

Comment:
Is widening a type a breaking change?

Reply:
@quantizor I just walk through changeset tutorial.
What should I put in instead of `major`? `minor` is ok?
Or could you write code suggestion here?

Reply:
I think probably just a `patch` is ok